conn_test.go

```diff
@@ -180,19 +145,11 @@ const (
 
 func init() {
 	var err error
-	serverKey, err := rsa.GenerateKey(rand.Reader, 2048)
-	if err != nil {
-		panic(err)
-	}
-	serverCert, err = newSelfSigned(serverName, RSA_PKCS1_SHA256, serverKey)
-	if err != nil {
-		panic(err)
-	}
-	clientKey, err := rsa.GenerateKey(rand.Reader, 2048)
+	serverKey, serverCert, err = MakeNewSelfSignedCert(serverName, RSA_PKCS1_SHA256)
```

Comment:
![Y U NO ECDSA](http://memedad.com/memes/1319411.jpg)

Reply:
I can change to ECDSA. The existing code had RSA, so I was just trying to avoid changing the behavior

Reply:
@bifurcation: just LMK